### DISCUSSION TITLE
Another country log in

### DISCUSSION THREAD

**Discussion by Moerte** [nVuln/homebridge-lg-thinq#51] [Q&A]
Is there a way to login on a LG Login page from another country?

**Comment by nVuln** [comment]
I don't understand, you can change country in plugin setting

**Reply by Moerte** [reply]
Forget it, my account was linked to Apple and in that log in page we can't log in with Apple from another country, even with that setting changed. 
I deleted and created a new one and solved the problem.

Thanks for answering
